paywall/next.config.js

```diff
@@ -45,6 +45,7 @@ module.exports = withSourceMaps({
 
     return {
       '/': { page: '/' },
+      about: { page: 'about' },
```

Comment:
Shouldn't this be `/about`?

Reply:
And: we already have an `about` page. As this will be shown as the landing page, could this be called `landing` or something unambiguous?